marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/impl/security.sjs

```diff
@@ -64,7 +64,10 @@ class Security {
     }
 
     if (currentRoleNames.includes("hub-central-downloader")) {
-      response.authorities.push("canDownloadConfigurationFiles");
+      response.authorities.push("downloadConfigurationFiles");
```

Comment:
Is this just a name refactor to the download story already merged?

Reply:
Right - I had pointed out to @ryanjdew that it's redundant to include the word "can". 

I'm also hoping for a day where we can dynamically build authorities based on patterns in role and privilege names, and we're not likely to have the word "can" in either of those names.